src/components/mission/Missionitem.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Badge from 'react-bootstrap/Badge';
+import { useDispatch } from 'react-redux';
+import { reservedMission, cancelMission } from '../../redux/missions/missionSlice';
+
+function MissionItem({ item }) {
```

Comment:
While you did a good job working on the project requirements I notices you disabled eslint. Its against microverse policy to disable linters because linters help us write clean code. It would be nice if you kindly enable eslint.